### ISSUE TITLE
when I use the eavesdrop funcion,after the call , the channels not be cleared.

### ISSUE THREAD

**Issue by WZHCM** [signalwire/freeswitch#1130]
Hello everyone, I've got a problem when I use the freeswitch（The version is 1.10.6-dev+git~20201109T06235Z）
Make a call from A to B with audio and video, then the device C enter the number to monitor (dialplan as below)
If any devices hand up, the call ended.(The whole process is ok）But after that I found the channels not be cleared.(screenshot as below)
When I try to use the device A or B to call , the audio is ok but no video, after hupall the video is also ok.
becaue the whole process is ok, so the logs do not record the problem.
It's probabilistic, triggered every 10 times. when it triggered, if I do not clear, no video during the calling.
How can I improve it ?

![1](https://user-images.githubusercontent.com/34757448/111933806-1ec87000-8ab8-11eb-813d-bd4dd0c9920d.png)
![2](https://user-images.githubusercontent.com/34757448/111933817-22f48d80-8ab8-11eb-92e6-3c0bb7c0ad7e.png)
![3](https://user-images.githubusercontent.com/34757448/111933823-2425ba80-8ab8-11eb-93c4-3fa7396a8571.png)

**Comment by andywolk** [comment]
@WZHCM Have you tried the latest master?

**Comment by WZHCM** [comment]
> @WZHCM Have you tried the latest master?

Yes,I have.  But this issue still happen after test many times.  Now I think I need a solution to solve it
